app/test-utils/testFunctions/commonComponentsTests.js

```diff
@@ -49,18 +49,18 @@ const submitButtonTest = ({ pageSetup, sectionManifest }) => {
   });
 };
 
-const submitButtonClickedTest = ({
+const submitButtonClickedTest = async ({
   pageSetup,
   sectionId,
   data,
   sectionManifest,
   dashboardId,
 }) => {
   if (sectionManifest.successfulSubmitResponsePath) {
-    test(`should go to ${sectionManifest.successfulSubmitResponsePath} when clicking the submit button`, async (t) => {
+    await test(`should go to ${sectionManifest.successfulSubmitResponsePath} when clicking the submit button`, async (t) => {
       await pageSetup({ t, responseBody: data });
       nock(apiLocalhost)
-        .put(`${apiPath}/sections/${sectionId}`)
+        .put(`${apiPath}/sections/${sectionId}`, data)
```

Comment:
the request body is the same as the response body?

Reply:
errrrrr yeah this doesn't look right

Reply:
so this is correct as just above we send in the data to populate the component so when submit is pressed again it just resubmits what is there ie the prepopulated data